roles/system/tasks/main.yml

```diff
@@ -54,14 +37,10 @@
   when: ansible_lsb.id == "Debian"
   tags: ['master','update']
 
-- name: Run apt-get update  
-  shell: apt-get update ; echo
-  register: result
-  tags: ['master','custom','update']
-
-- name: Fix broken package
-  shell: dpkg --configure -a
-  when: "'dpkg --configure -a' in result.stderr"
+- name: Update APT cache, not more than once every 8 hours
+  apt: 
+    update_cache: yes 
+    cache_valid_time: 28800
```

Comment:
Ideascube role depend on that apt-get update refresh now. We will have to remember that